android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1664,7 +1677,15 @@ public void onVolumeChanged(float volume) {
 
     @Override
     public void onIsPlayingChanged(boolean isPlaying) {
-        eventEmitter.playbackStateChanged(isPlaying);
+        if (isPlaying && isSeeking) {
+            eventEmitter.seek(player.getCurrentPosition(), seekPosition);
+        }
+
+        eventEmitter.playbackStateChanged(isPlaying, isSeeking);
+
+        if (isPlaying) {
```

Comment:
this test should be before  eventEmitter.playbackStateChanged ?
becuase you are changing isSeeking, but not sending it to app ?! 🤔

Reply:
Yes that's correct the flow is:
1. when seek "is detected" `onPositionDiscontinuity` is called and in it we are setting `isSeeking` to true
2. after seeking is done (and we want to triger `onSeek` on JS side) two thing can happen:
 A) If player was not paused when user seek -> `onIsPlayingChanged` will be called
 B) If player was paused when user seek -> we will catch that loading is finished in `onBuffering`
 
We are sending event with `isSeeking=true` bcs we want to inform that `onPlaybackStateChanged` was called bcs of seeking, and then we will set in back to `false`

This is needed to check if we should show spiner when user is seeking (custom controls) - otherwise we don't know if `onPlaybackStateChanged` was called bcs of "pause/resume" or "seeking"

So I believe this is correct